cypress/.eslintrc.js

```diff
@@ -0,0 +1,12 @@
+module.export = {
+  root: true,
+  plugins: [
+    "eslint-plugin-cypress"
+  ],
+  extends: [
+    "plugin:cypress/recommended"
+  ],
+  env: {
+    "cypress/globals": true
+  },
+};
```

Comment:
adding a separate linting file for cypress in it's directory or not you can define at root in the eslint file as well